helpers/compile/plugins/fill-plugin/fillers/perf_hooks.ts

```diff
@@ -0,0 +1,4 @@
+const performance = globalThis['performance']
```

Comment:
Was not needed before because OTL packages in `engines` core had different version, which did not call `performance.now()`. Will be irrelevant again once we stop bundling OTL packages in the client.